### ISSUE TITLE
MetricExpansionConfig not working as expected

### ISSUE THREAD

**Issue by fabiosf** [librato/metrics-librato#54]
Hi,

I'm using "metrics-librato" version "4.0.1.6" to send some "Coda Timers" attributes (version 3.0.2) to Librato, but find some unexpected behaviour which may be a bug.

"metrics-librato" docs describes "MetricExpansionConfig" as a way to reduce the number of metrics sent to Librato.

I've followed the docs, but found that "MetricExpansionConfig" does not filter some asked metrics. In my case, I asked to send only "p999" and "p95" Coda Timer's attributes, but reporter is sending "mean" as well.

So, in Librato, I'm getting 3 metrics for each "Coda Timer" instead of just 2, like the following example:
1. *software_metrics.rdbms.getDF*
2. **software_metrics.rdbms.getDF.95th**
3. **software_metrics.rdbms.getDF.999th**

Note that (1) is the "mean" and should not be there. (2) and (3) were correctly sent to Librato as expected.

The code that I used to found this is ahead.

```java
//Mark which metrics will be sent to Librato:
Set<ExpandedMetric> metricsToBeSentToLibrato = new HashSet<ExpandedMetric>();
metricsToBeSentToLibrato.add(ExpandedMetric.PCT_95);
metricsToBeSentToLibrato.add(ExpandedMetric.PCT_999);

Builder libratoReporterBuilderFrontendWeb = 
LibratoReporter.builder(UtilsMetrics.getMetricsRegistry(), "it@companyname.com.br", "0c76cf5aea0d982d18b4615763cea28fc1b9f1f48a81199b7321c516b3e6a225", UtilsMetrics.getHostname())
.setDurationUnit(TimeUnit.MILLISECONDS)
.setRateUnit(TimeUnit.SECONDS)
.setTimeoutUnit(TimeUnit.MILLISECONDS)
.setTimeout(5000)
.setTimeout(5000, TimeUnit.MILLISECONDS)
.setExpansionConfig(new MetricExpansionConfig(metricsToBeSentToLibrato));

LibratoReporter.enable(libratoReporterBuilderFrontendWeb, 10, TimeUnit.SECONDS);
```

Am I doing something wrong or is there a way to filter unneeded attributes on "metrics-librato" version "4.0.1.6"?

Thanks
Fabio

**Comment by collinvandyck** [comment]
Thanks for reporting your issue. I've merged in a change for this, and will be making a release to maven-central shortly.

**Comment by collinvandyck** [comment]
I've just released `4.0.1.7` which should be available on Maven Central shortly. Check the project readme for information on how to enable this. Thanks!
